model_analyzer/triton/client/client.py

```diff
@@ -174,3 +180,20 @@ def is_server_ready(self):
         Returns true if the server is ready. Else False
         """
         return self._client.is_server_ready()
+
+    def _check_for_triton_log_errors(self, log_file):
```

Comment:
wondering if this responsibility should be on the server class and not the client class to parse and understand the server output. Weird that the client is reaching in to the server log and making decisions on it. Then again not sure how to make that happen without passing the server object into the client.

Reply:
I thought it was cleaner this way, else you have to pass the entire sever class into the client and then call a method there that checks the output.